### PR TITLE
M2-1134 Item question

### PR DESCRIPTION
**Pull request by wildkiomi**
Screenshot from 2023-04-07 13-02-55

### CODE REVIEW

File: src/modules/Dashboard/features/RespondentData/CollapsedMdText/CollapsedMdText.styles.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  max-height: ${({
+    maxHeight,
+    isOpen,
+    isLarge,
+  }: {
+    maxHeight: number;
+    isOpen: boolean;
+    isLarge: boolean;
+  }) => (!isOpen ? `${maxHeight}px` : 'unset')};
+  border-bottom: ${({ isOpen, isLarge }: { isOpen: boolean; isLarge: boolean }) =>
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/CollapsedMdText/CollapsedMdText.styles.ts:28]: { isOpen: boolean; isLarge: boolean } could be removed from here, they have already declared at 25, 26 rows

File: src/modules/Dashboard/features/RespondentData/CollapsedMdText/CollapsedMdText.styles.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { Button, styled } from '@mui/material';
+import { Box } from '@mui/system';
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/CollapsedMdText/CollapsedMdText.styles.ts:2]: you could also import Box from mui/material

**Review by IhorNesterenko99 — APPROVED** [review]

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/Feedback.styles.ts [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-import { StyledClearedButton, theme, variables } from 'shared/styles';
+import { StyledClearedButton, StyledFlexColumn, theme, variables } from 'shared/styles';
 
-export const StyledContainer = styled(Box)`
+export const StyledContainer = styled(StyledFlexColumn)`
   width: 44rem;
   height: 100%;
   position: absolute;
   right: 0;
+  z-index: 2;
```
Comment (alex-i-e) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/Feedback.styles.ts:10]: is it possible to use one of `theme.zIndex` value ?

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.styles.ts [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
   overflow-y: auto;
 `;
 
-export const StyledFeedbackBtn = styled(StyledTextBtn)`
-  position: absolute;
-  top: 2rem;
-  right: 8rem;
+export const StyledHeader = styled(StyledFlexTopCenter, shouldForwardProp)`
+  ${commonStickyStyles};
+  z-index: 1;
```
Comment (alex-i-e) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.styles.ts:15]: is it possible to use one of `theme.zIndex` value ?

**Review by muhurov — APPROVED** [review]

**Pull request merged**